central/complianceoperator/v2/checkresults/stats/service/service_impl_test.go

```diff
@@ -869,6 +881,30 @@ func (s *ComplianceResultsStatsServiceTestSuite) TestGetComplianceProfileCheckSt
 	}
 }
 
+func getExpectedControlResults() []*ruleDS.ControlResult {
+	return []*ruleDS.ControlResult{
+		{RuleName: "rule-name", Standard: "OCP-CIS", Control: "1.2.2"},
+		{RuleName: "rule-name", Standard: "OCP-CIS", Control: "1.3.3"},
+		{RuleName: "rule-name", Standard: "OCP-CIS", Control: "1.4.4"},
+	}
+}
+
+func getExpectedBenchmark() []*storage.ComplianceOperatorBenchmarkV2 {
```

Comment:
[nit] Would it make sense to have these in a separate package (like `testutils`) since it's the same function in the other tests?